File: IncAng/src/dto/LinkTKDTO.ts
import { ThingDTO } from '../dto/ThingDTO';
import { KeywordDTO } from '../dto/KeywordDTO';

export class LinkTKDTO {
    idLinkTK: number;
    thing: ThingDTO;
    keyword: KeywordDTO;
    
    constructor(id:number, thing: ThingDTO, keyword: KeywordDTO){
        this.idLinkTK = id;
        this.thing = thing;
        this.keyword = keyword;
    }
}

File: src/main/java/it/contrader/dto/LinkTKDTO.java
package it.contrader.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Data
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class LinkTKDTO {
	private Integer idLinkTK;
	private ThingDTO thing;
	private KeywordDTO keyword;
}
